src/py/flwr/simulation/ray_transport/utils.py

```diff
@@ -55,3 +60,25 @@ def enable_tf_gpu_growth() -> None:
             log(ERROR, traceback.format_exc())
             log(ERROR, ex)
             raise ex
+
+
+def check_clientfn_returns_client(client: Client) -> Client:
+    """Warn once that clients returned in `clinet_fn` should be of type Client.
```

Comment:
Are we sure this will be displayed only once? It looks like it will be run every time `VirtualClientEngineActor.run` runs.

Reply:
yes. once each actor runs a client_fn for the first time.